T5474: remove "aws glb" XML definitions as feature is not available in VyOS 1.4

c-po opened the pull request:
<!-- All PR should follow this template to allow a clean and transparent review -->
<!-- Text placed between these delimiters is considered a comment and is not rendered -->

## Change Summary
<!--- Provide a general summary of your changes in the Title above -->

Files got added by accident in commit c9eaafd9f ("T5474: establish common file name pattern for XML conf mode commands")

## Types of changes
<!---
What types of changes does your code introduce? Put an 'x' in all the boxes that apply.
NOTE: Markdown requires no leading or trailing whitespace inside the [ ] for checking
the box, please use [x]
-->
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)
<!-- All submitted PRs must be linked to a Task on Phabricator. -->
* https://vyos.dev/T5261

## Related PR(s)
<!-- Link here any PRs in other repositories that are required by this PR -->
* https://github.com/vyos/vyos-1x/pull/2185
* https://github.com/vyos/vyos-1x/pull/2730

## Component(s) name
<!-- A rather incomplete list of components: ethernet, wireguard, bgp, mpls, ldp, l2tp, dhcp ... -->
AWS GLB

## Proposed changes
<!--- Describe your changes in detail -->

## How to test
<!---
Please describe in detail how you tested your changes. Include details of your testing
environment, and the tests you ran. When pasting configs, logs, shell output, backtraces,
and other large chunks of text, surround this text with triple backtics
```
like this
```
-->



## Smoketest result
<!-- Provide the output of the smoketest
```
$ /usr/libexec/vyos/tests/smoke/cli/test_xxx_feature.py
test_01_simple_options (__main__.TestFeature.test_01_simple_options) ... ok
```
-->

```
cpo@LR2.wue3:~$ /usr/libexec/vyos/tests/smoke/cli/test_configd_inspect.py
test_config_modification (__main__.TestConfigdInclude.test_config_modification) ... ok
test_file_instance (__main__.TestConfigdInclude.test_file_instance) ... ok
test_function_instance (__main__.TestConfigdInclude.test_function_instance) ... ok
test_signatures (__main__.TestConfigdInclude.test_signatures) ... ok

----------------------------------------------------------------------
Ran 4 tests in 1.091s

OK
```

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
<!--- The entire development process is outlined here: https://docs.vyos.io/en/latest/contributing/development.html -->
- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [x] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

sever-sever commented:
There also need other files like `systemd` https://github.com/vyos/vyos-1x/pull/2185/files#diff-361f8a50d0a99a0690159a34bc779c75296c96b250e75e62c713b39dc198c646

c-po committed:
T5474: remove "aws glb" XML definitions as feature is not available in VyOS 1.4

Files got added by accident in commit c9eaafd9f ("T5474: establish common file
name pattern for XML conf mode commands")

c-po commented:
> There also need other files like `systemd` https://github.com/vyos/vyos-1x/pull/2185/files#diff-361f8a50d0a99a0690159a34bc779c75296c96b250e75e62c713b39dc198c646

Yeah something more is missing, we need to do a full backport. This PR just fixes Smoketests
